TELCODOCS-2215: Clarifying extra manifest order of application (#89296)
openshift/openshift-docs: modules/cnf-image-based-upgrade-prep-extramanifests.adoc

Create additional manifests that you want to apply to the target cluster.

[NOTE]
====

> If you add more than one extra manifest, and the manifests must be applied in a specific order, you must prefix the filenames of the manifests with numbers representing the order in which they should be applied. For example, `00-namespace.yaml`, `01-sriov-extra-manifest.yaml`, and so on.

Reviewer: ```suggestion
If you add more than one manifest, to apply the manifests in a specific order, you must prefix the filenames of the manifests with numbers in the order you want to apply them. For example, `00-namespace.yaml`, `01-sriov-extra-manifest.yaml`, and so on.
```

- Suggested a change to avoid should. See: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#s
- Suggested to _replace/passive-voice/active-voice_

Author: Thanks! I reworded to avoid "should".
